pkgs/development/tools/symfony-cli/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation rec {
```
stdenvNoCC can probably be used here. Please test this.